analyzer/src/main/kotlin/managers/Pub.kt

```diff
@@ -169,6 +193,62 @@ class Pub(
 
     override fun getVersionRequirement(): Requirement = Requirement.buildIvy("[2.2,)")
 
+    // Bootstrap flutter if it's not yet installed
+    override fun beforeResolution(definitionFiles: List<File>) {
+        if (getPathFromEnvironment(flutterCommand) != null) {
+            log.info { "Skipping to bootstrap flutter as it is already in PATH." }
+            return
+        }
```

Comment:
Does it really make sense to keep this check, given that we also have the check below based on `flutterAbsolutePath`, which might also refer to `getPathFromEnvironment()`?